apps/admin/frontend/src/components/smartcard_modal/program_election_card_view.tsx

```diff
@@ -105,21 +104,17 @@ export function ProgramElectionCardView({
               >
                 Poll Worker Card
               </Button>
-            </p>
-            <HorizontalRule />
+            </P>
```

Comment:
What do you think about getting rid of the "or"? I feel like it's generally implicit that the user has a choice between two buttons that are next to each other.

Reply:
+1 - never really liked those and have seen a bunch of instances in various apps. Can update them in a cleanup pass later - just keeping this round of PRs to mostly component replacements

Reply:
Yay, thanks!